### ISSUE TITLE
An issue with Botulus

### ISSUE THREAD

**Issue by lechaz79** [mtgred/netrunner#7289]
Hi all! First of all, thank you for Jnet and all the fun I have with it.

Now, here is a bug I encountered. During a run, I encountered a Brân 1.0 with a Botulus and 3 tokens on it. I tried to break all the subroutines, but somehow I broke the same subroutines twice and the button to break the last subroutine disappeared. Meanwhile, there was still a token left on Botulus but I was unable to use it. Look at the screenshot I took.
![Capture d'écran 2024-02-18 202833](https://github.com/mtgred/netrunner/assets/52366164/75017075-4d50-419d-8f1a-c48dcdf88f21)

**Comment by NoahTheDuke** [comment]
I believe that's merely an issue with how it was displayed. The top subroutine is just too long to display the red x adjacent. I'm not sure about the counter staying on botulus, that will require some effort to untangle.

**Comment by NoahTheDuke** [comment]
Yeah, something fucky has happened, either the state got out of sync or an effect was applied in the wrong order. I'll write a test, see if I can re-create it.

**Comment by lechaz79** [comment]
Actually, I saved a replay of this game. Would it be helpful?

Envoyé depuis Proton Mail mobile

-------- Message d'origine --------
